apps/meteor/server/models/raw/LivechatContacts.ts

```diff
@@ -195,4 +199,36 @@ export class LivechatContactsRaw extends BaseRaw<ILivechatContact> implements IL
 			options,
 		).toArray();
 	}
+
+	isContactActiveOnPeriod(visitorId: string, period: string): Promise<boolean> {
+		const query = {
+			'channels.visitorId': visitorId,
+			'activity': period,
```

Comment:
Is this query using the index? I kinda remember pfe indexes being a bit picky on when to cover a query

Reply:
Yeppp, Compass tells me it's being used :candle: